### DISCUSSION TITLE
use standard library function for nan("") in datatype module

### DISCUSSION THREAD

**Discussion by 00thirdeye00** [cieslarmichal/faker-cxx#814] [General]
https://github.com/cieslarmichal/faker-cxx/blob/540cea831c11b5680454a148276388fd5e201a84/src/modules/datatype/datatype.cpp#L16

this line will always be true. instead it might be better if we use standard library function to check if a number is `nan` using `isnan(x)`

**Comment by cieslarmichal** [comment]
agreed

**Reply by 00thirdeye00** [reply]
I will create this as new issue
